app/models/move_event.rb

```diff
@@ -1,8 +1,12 @@
 class MoveEvent < Event
   # NB: this class exposes a few methods specific to moves to the Event model
 
+  def create_in_nomis?
```

Comment:
can you add a test for this? (although rest of those aren't tested previously 🤔 )

Reply:
Discussed offline - has coverage via the move runner specs for true/false scenarios.